cmd/vic-machine/configure/configure.go

```diff
@@ -45,6 +47,24 @@ type Configure struct {
 
 	upgrade  bool
 	executor *management.Dispatcher
+
+	regenerateCerts bool
+	scert           string
```

Comment:
just include the whole certificate structure here, instead of duplicate variable definition